content/docs/command-reference/add.md

```diff
@@ -144,6 +144,11 @@ directory symlinks.
 - `--external` - allow `targets` that are outside of the DVC repository. See
   [Managing External Data](/doc/user-guide/managing-external-data).
 
+- `--glob` - searches for pathnames matching the patterns specified by the
+  `targets`, according to rules used by the shell. It supports the Unix
+  shell-style wildcards `*`, `?`, `[seq]`, `[!seq]`, and the recursive glob
+  `**`. The order of the results depends on the file system.
```

Comment:
what is the order of results from the `dvc add` perspective?

Reply:
yeah, it makes no sense. I was focusing on the functionality behind it. I will correct it.